File: app/src/main/java/com/scatl/uestcbbs/module/search/view/SearchView.java
package com.scatl.uestcbbs.module.search.view;

import com.scatl.uestcbbs.entity.SearchPostBean;
import com.scatl.uestcbbs.entity.SearchUserBean;

public interface SearchView {
    void onSearchUserSuccess(SearchUserBean searchUserBean);
    void onSearchUserError(String msg);
    void onSearchPostSuccess(SearchPostBean searchPostBean);
    void onSearchPostError(String msg);
}

File: app/src/main/java/com/scatl/uestcbbs/module/search/view/SearchView.kt
package com.scatl.uestcbbs.module.search.view

import com.scatl.uestcbbs.base.BaseView
import com.scatl.uestcbbs.entity.SearchPostBean
import com.scatl.uestcbbs.entity.SearchUserBean

/**
 * Created by sca_tl at 2023/2/8 10:22
 */
interface SearchView: BaseView {
    fun onSearchUserSuccess(searchUserBean: SearchUserBean)
    fun onSearchUserError(msg: String?)
    fun onSearchPostSuccess(searchPostBean: SearchPostBean)
    fun onSearchPostError(msg: String?)
}